src/client/trackers/real-user-monitoring-for-performance.js

```diff
@@ -1,60 +1,67 @@
-import ttiPolyfill from 'tti-polyfill';
+import Perfume from 'perfume.js';
 import { broadcast } from '../broadcast';
 import { userIsInCohort } from '../utils/userIsInCohort';
 
+const cohortPercent = 5;
 export const realUserMonitoringForPerformance = () => {
-	const cohortPercent = 5;
 	if (!userIsInCohort(cohortPercent)) return;
 
-	// For browser compatibility @see: https://mdn.github.io/dom-examples/performance-apis/perf-api-support.html
-	if (!'PerformanceLongTaskTiming' in window || !'ttiPolyfill' in window) return;
+	const navigation = performance.getEntriesByType('navigation')[0];
```

Comment:
I think a safety check should be added here to bail early if the browser does not support `PerformanceLongTaskTiming`.

Reply:
That safety check is in Perfume.js, but happy to add it here too. What do you think?

Reply:
I think this code is run before Perfume is initialised so it could still go 💥

Reply:
It is, and it could. Guardian added. Cheers :)